Gruntfile.js

```diff
@@ -392,19 +418,22 @@ module.exports = function(grunt) {
                   echo 'ERROR: Links found with target="_blank" but no rel="noopener noreferrer" set.  Please add required rel attribute.')`,
       },
       'setup-admin': {
+        stderr: true,
```

Comment:
Is this deliberate?

Reply:
Don't remember what that is from actually ... Removed it